src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -861,6 +861,15 @@ private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMe
                 CheckForImplementationOfCorrespondingPropertyOrEvent((MethodSymbol)interfaceMember, implementingType, implementingTypeIsFromSomeCompilation, ref implicitImpl);
             }
 
+            if ((object)implicitImpl == null && (object)closestMismatch == null)
+            {
+                // Check for default interface implementations
+                if (!interfaceMember.IsAbstract)
+                {
+                    implicitImpl = interfaceMember;
+                }
+            }
+
```

Comment:
This is correct only for the simplest scenario. This doesn't take into account interface member overrides appearing in interfaces. Please add a PROTOTYPE comment.